src/main/java/gov/ornl/ssm/ml/ui/views/MainScreen.java

```diff
@@ -24,10 +31,21 @@ public MainScreen() {
 		//Links to each capability
 		RouterLink trainingLink = new RouterLink("Train a New Filter", TrainingView.class);
 		RouterLink classifyLink = new RouterLink("Classify a Sample", ClassificationView.class);
+		Anchor logout = new Anchor("logout", "Logout");
 		
 		//Layout
 		add(trainingLink);
 		add(classifyLink);
+		add(logout);
+		
+		//Demonstration of how to access OIDC info
+//		Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+//		if (authentication.getPrincipal() instanceof OidcUser) {
+//		    OidcUser principal = ((OidcUser) authentication.getPrincipal());
+//		    add(new Label(principal.getGivenName()));
+//		}
+//		add(new Label(authentication.getPrincipal().toString()));
+//		add(new Label(authentication.getPrincipal().getClass().getCanonicalName()));
```

Comment:
Any reason to keep this commented code in?

Reply:
No. Removed in the latest commit.